torch/backends/cuda/__init__.py

```diff
@@ -207,6 +208,57 @@ def preferred_linalg_library(
     return torch._C._get_linalg_preferred_backend()
 
 
+_BlasBackends = {
+    "default": torch._C._BlasBackend.Default,
```

Comment:
No need to add default here as we don't have any heuristics to choose between LT and cublas. This should be `cublas`, as the docs below announce.

Reply:
Done.